androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -142,6 +146,7 @@ fun EnhancedVideoPlayer(
                 currentTime = player.contentPosition
                 totalDuration = player.duration
                 loop = player.repeatMode == ExoPlayer.REPEAT_MODE_ALL
+                deviceVolume = player.deviceVolume
```

Comment:
Will onEvents trigger when the user changes the volume from device button?
Check if onDeviceVolumeChanged do it for you (I don't know if trigger just if the exoplayer attribute is changed in the app). If not, you need to find out another way to do that.

Reply:
[answered on slack](https://profusion-mobi.slack.com/archives/C05EQJ16PHA/p1688498537672149?thread_ts=1688079385.365879&cid=C05EQJ16PHA)

Reply:
Yeah, it is triggering.